### PR TITLE
Fix some doc typos

### PR DESCRIPTION
**Pull request by pinicarus**

### CODE REVIEW

**Comment by pinicarus** [comment]
Hi,

This is my first PR on Terra, since I was reading the doc to learn more about it.

The project looks really great. I still need to learn a bit more about it but I hope to use it quite soon.

File: docs/api.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ True if `type` is a pointer to a struct.
 
 ---
 
-    function types.type:ispointertofunction()
+    types.type:ispointertofunction()
```
Comment (elliottslaughter) [inline docs/api.md:404]: should this actually be `type:ispointertofunction()` (i.e. no `types.`)

**Review by elliottslaughter — APPROVED** [review]
Thanks, these changes look good! I only spotted one minor issue that could be fixed, and can merge once that's applied.

File: docs/api.md (hunk 1 above)
Comment (pinicarus) [reply docs/api.md]: Yes, I concur and changed it

**Comment by elliottslaughter** [comment]
Thank you!

**Pull request merged**